modules/generator/instance.go

```diff
@@ -184,6 +185,11 @@ func (i *instance) updateProcessors() error {
 		return err
 	}
 
+	i.ingestionSlackOverride = i.overrides.MetricsGeneratorIngestionSlack(i.instanceID)
+	if i.ingestionSlackOverride == 0 {
```

Comment:
i hate that we do this everywhere. i wish the overrides system integrated the config and returned the correct result if it was unset.

nothing to fix in this PR tho :)